module/attribute/src/Infrastructure/Validator/AttributeGroupCodeValidator.php

```diff
@@ -43,7 +43,7 @@ public function validate($value, Constraint $constraint): void
 
         $value = (string) $value;
 
-        if (!\Ergonode\Attribute\Domain\ValueObject\AttributeGroupCode::isValid($value)) {
+        if (!\Ergonode\Attribute\Domain\ValueObject\AttributeGroupCode::isValid(strtolower($value))) {
```

Comment:
I'm not sure that problem is here, as we look into the class :
```
namespace Ergonode\Attribute\Domain\ValueObject;

class AttributeGroupCode extends AbstractCode
{
    ...
    public static function isValid(string $value): bool
    {
        $value = strtolower($value);

        return parent::isValid($value)
            && preg_match(self::PATTERN, $value);
    }
}
```
there is already used `strtolower` function inside a `isValid` method

Reply:
Are we looking into the same branch @rprzedzik ?
https://github.com/ergonode/backend/blob/develop/module/attribute/src/Domain/ValueObject/AttributeGroupCode.php#L38

Reply:
we no longer work on develop branch is outdated, work on master branch or on specific lates tag.
If somewhere in the documentation there was information about the development branch, please give us the place, we have corrected it 
Sorry for the difficulty.